pkg/infrastructure/kogito_service.go

```diff
@@ -17,13 +17,18 @@ package infrastructure
 import (
 	"fmt"
 	"net/url"
+	"os"
 
 	"github.com/kiegroup/kogito-cloud-operator/pkg/apis/app/v1alpha1"
 	"github.com/kiegroup/kogito-cloud-operator/pkg/client"
 	"github.com/kiegroup/kogito-cloud-operator/pkg/client/kubernetes"
 	"github.com/kiegroup/kogito-cloud-operator/pkg/framework"
 )
 
+const (
+	envVarKogitoServiceURL = "KOGITO_SERVICE_URL"
```

Comment:
this same var is set on  pkg/controller/kogitoruntime/deployer.go , can't we set it on another place so it can be used by both classes?

Reply:
I believe the purpose of this variable has a different meaning in this context. This one is supposed to be set in scenarios where we run the operator locally, so the `KOGITO_SERVICE_URL` would be the exposed route or the minikube URL.

Reply:
As @ricardozanini mentioned, this one is a local environment variable for running the operator locally. The other one in pkg/controller/kogitoruntime/deployer.go is an internal environment variable inside the container.

Reply:
maybe rename is enough.